ocs_ci/utility/ibmcloud.py

```diff
@@ -1242,6 +1242,43 @@ def find_free_network_subnets(subnet_cidr, network_prefix=27):
             return possible_subnets
 
 
+def delete_dns_records(cluster_name):
+    """
+    Delete DNS records leftover from cluster destroy.
+
+    Args:
+        cluster_name (str): Name of the cluster, used to filter DNS records
+
+    """
+    dns_domain_id = constants.IBMCLOUD_BASE_DOMAIN_ID
+    cis_instance_name = constants.IBMCLOUD_CIS_INSTANCE_NAME
+    ids_to_delete = []
+    page = 1
+    while True:
+        out = run_ibmcloud_cmd(
+            f"ibmcloud cis dns-records {dns_domain_id} --per-page 1000 --page {page} --output json"
```

Comment:
Btw. shouldn't be the `-i {cis_instance_name}` argument used also in this command?

Because when I tried to run this command locally, it failed with following error:
```
$ ibmcloud cis dns-records a180c88cc263d58808a739e1fdfc268a --per-page 100 --page 1
FAILED
Context instance is not set yet. Use 'ibmcloud cis instance-set INSTANCE' to set the context instance before running commands.
```

Reply:
We need to first run this command:
```
ibmcloud cis instance-set odf-qe-cis
```

And once this is set it will allow us to run CIS related operations - this has to be triggered only once for the session, so we should run this outside of the while loop - somewhere before while loop perheps.

I think I shared this command with you yesterday in DM Coady.

Reply:
Good catch, this is being explicitly set now.